pennylane/ops/qutrit/non_parametric_ops.py

```diff
@@ -430,3 +430,163 @@ def pow(self, z):
+            label += "⁻¹"
+
+        return label
+
+    def __init__(
+        self, wires, subspace=None, do_queue=True
+    ):  # pylint: disable=dangerous-default-value
+        if subspace is not None and not hasattr(subspace, "__iter__"):
+            raise ValueError(
+                "The subspace must be a sequence with two unique elements from the set {0, 1, 2}."
+            )
+
+        self._subspace = subspace
+        self._hyperparameters = {
+            "subspace": self.subspace,
+        }
+        super().__init__(wires=wires, do_queue=do_queue)
+
+    @property
+    def subspace(self):
+        """The single-qutrit basis states which the operator acts on
+
+        This property returns the 2D subspace on which the operator acts. This subspace
+        determines which two single-qutrit basis states the operator acts on. The remaining
+        basis state is not affected by the operator.
+
+        Returns:
+            tuple[int]: subspace on which operator acts
+        """
+        return tuple(sorted(self._subspace)) if self._subspace is not None else None
+
+    @staticmethod
+    def compute_matrix(subspace=None):  # pylint: disable=arguments-differ
+        r"""Representation of the operator as a canonical matrix in the computational basis (static method).
+
+        The canonical matrix is the textbook matrix representation that does not consider wires.
+        Implicitly, this assumes that the wires of the operator correspond to the global wire order.
+
+        .. seealso:: :meth:`~.THadamard.matrix`
+
+        Args:
+            subspace (Sequence[int]): the 2D subspace on which to apply operation. This should be
+            `None` for the generalized Hadamard.
+
+        Returns:
+            ndarray: matrix
+
+        **Example**
+
+        >>> print(qml.TH.compute_matrix(subspace=[0, 2]))
+        array([[ 1.,  0.,  1.],
+               [ 0.,  1.,  0.],
+               [ 1.,  0., -1.]])
+        """
+
+        if subspace is None:
+            return (-1j / np.sqrt(3)) * np.array(
+                [[1, 1, 1], [1, OMEGA, OMEGA**2], [1, OMEGA**2, OMEGA]]
+            )
+
+        if not hasattr(subspace, "__iter__") or len(subspace) != 2:
+            raise ValueError(
+                "The subspace must be a sequence with two unique elements from the set {0, 1, 2}."
+            )
+
+        if not all(s in {0, 1, 2} for s in subspace):
+            raise ValueError("Elements of the subspace must be 0, 1, or 2.")
+
+        if subspace[0] == subspace[1]:
+            raise ValueError("Elements of subspace list must be unique.")
+
+        subspace = tuple(sorted(subspace))
+
+        mat = np.eye(3, dtype=np.complex128)
+
+        unused_ind = list({0, 1, 2}.difference(set(subspace))).pop()
+
+        mat[unused_ind, unused_ind] = np.sqrt(2)
+        mat[subspace[0], subspace[1]] = 1
+        mat[subspace[1], subspace[0]] = 1
+        mat[subspace[1], subspace[1]] = -1
+
+        return mat / np.sqrt(2)
```

Comment:
Okay given that the sqrt(2) is here I suspect that it is just missing in the example output?

Reply:
Yes, that was the issue. I've fixed the documentation now.